ducanhpham0312/zeevision
# feat: single instance page

### Linked issue

Closes ducanhpham0312/zeevision-private#77


### Description

- Implement `/instancee/:id` page with instance content and variables table
- Add corresponding navigation link to the `/instances`, `processes`, `processes/id` pages.
- Add new allowed type `ReactNode` to table content as we need to use `NavLink`.
- Add white background color to Table

### Testing

- Run test bench
- Run zeevision. Remember to `docker compose down -v` before `docker compose up --build`
- Login to pgadmin, create a server and copy-paste this query: https://codefile.io/f/7okeTlDomd
- Open the frontend `localhost:8080/processes`. Clicking an instance key should open the `instance/:id` page with the variables table
- Open `instances`/`process/:id`  page, clicking on the instance key should behave the same
- End time field is included in the instance vertical table.
